popSimManu.tex

```diff
@@ -667,14 +704,20 @@ \subsection*{Model QC procedures}
 into \stdpopsim that allows for rigorous quality control. Imagine Developer A
 wants to introduce a new model into \stdpopsim. Developer A implements the
 demographic model for the relevant organism along with clear documentation
-of the model parameters and populations. This model is submitted as a Pull Request,
+of the model parameters and populations. 
+% ALG: I’m not sure what documentation of populations means. 
+This model is submitted on GitHub as a Pull Request,
 where it is evaluated by a reviewer and then included as `preliminary',
 but is not linked to the online documentation nor the command line interface.
-Inclusion of a preliminary model triggers a QC issue, and a second developer,
+Inclusion of a preliminary model 
+% ALG: automatically?
```

Comment:
actually it's not automatic

Reply:
I feel like "triggers" implies automatic...
Maybe rephrase to active. Like "Developer A creates a QC issue"